scripts/unzip_backup.py

```diff
@@ -94,10 +96,10 @@ def push_csv_to_db(extracted_csv_path):
                     reader = csv.reader(f)
 
                     for row in reader:
-                        report_period_start = row[0].replace(" +0000 UTC", "")
-                        report_period_end = row[1].replace(" +0000 UTC", "")
-                        interval_start = row[2].replace(" +0000 UTC", "")
-                        interval_end = row[3].replace(" +0000 UTC", "")
```

Comment:
@skanthed Is there a need to remove the offset? I thought PostgreSQL accepts the format with offset.

Reply:
I thought I kept the offset. When I update the hostname that change was again replaced. I will fix it.

Reply:
Its fixed now.

Reply:
Is this change trying to include the UTC offset when parsing timestamps? By removing only substring ' UTC', the resulting time will have the form 'YYYY-MM-DD HH:MI:SS +0000'.